docs/mddocs/Quickstart/npu_quickstart.md

```diff
@@ -86,12 +86,21 @@ pip install --pre --upgrade ipex-llm[npu]
 
 ## Runtime Configurations
 
-For `ipex-llm` NPU support, set the following environment variable with active `llm-npu` environment:
+For `ipex-llm` NPU support, please set the following environment variable with active `llm-npu` environment based on your device:
+
+- For **Intel Core™ Ultra Processers (Series 2) with processor number 2xxV**
 
 ```cmd
 set BIGDL_USE_NPU=1
 
-:: [optional] for MTL support
+:: [optional] for Intel Core™ Ultra 5 Processor 228V & 226V
+set IPEX_LLM_NPU_DISABLE_COMPILE_OPT=1
+```
```

Comment:
It's not optional for 228V & 226V; make two bullets for 228V & 226V, and 258V respectively.

Reply:
> It's not optional for 228V & 226V; make two bullets for 228V & 226V, and 258V respectively.

Have updated.